java/java.lsp.server/vscode/src/testAdapter.ts

```diff
@@ -71,7 +74,9 @@ export class NbTestAdapter {
                     }
                 }
             }
-            this.itemsToRun.forEach(item => this.set(item, 'skipped'));
+            if (this.started) {
```

Comment:
Shouldn't be `cancellation` checked also in the 1st branch (line ~65)?

Reply:
Yes this change prevents tests to be marked as skipped in case they are not even started (e.g. due to a compilation error).
Code modified to check `cancellation` in the first branch.